### PR TITLE
Add comment for clusters and UI display

### PR DESCRIPTION
**Pull request by BlackBear2003**
## What's the purpose of this PR

Add comment for clusters and UI display

## Which issue(s) this PR fixes:
Fixes #https://github.com/apolloconfig/apollo/issues/5067

## Brief changelog

it looks like: 

WX20240108-215908@2x
image

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Read the [Contributing Guide](https://github.com/apolloconfig/apollo/blob/master/CONTRIBUTING.md) before making this pull request.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit tests to verify the code.
- [ ] Run `mvn clean test` to make sure this pull request doesn't break anything.
- [ ] Update the [`CHANGES` log](https://github.com/apolloconfig/apollo/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by nobodyiam** [comment]
This looks great! Please find some comments below:
1. I think the comment is similar to the `remarks` of namespace, so it's better to use the same term, e.g. `Remarks` and `备注`
image
image

2. I think the comment should not be mandatory, as it's not a necessary information
3. When displaying the comment, it's not necessary to display the prefix `comment: `

**Comment by BlackBear2003** [comment]
> I think the comment is similar to the remarks of namespace, so it's better to use the same term, e.g. Remarks and 备注

changed terms at front-end from 'comment' to 'remarks'.

I'm not sure should we change the sql and entity's field name? I see there is `comment` in AppNamespace, Commit and Item while there is not `remarks`. So I just use `remark` when UI display.

image
image

File: apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/entity/Cluster.java [hunk 1]
```diff
@@ -67,9 +70,17 @@ public void setParentClusterId(long parentClusterId) {
...
+  }
+
+  public void setComment(String comment) {
+    this.comment = comment;
+  }
+
   public String toString() {
     return toStringHelper().add("name", name).add("appId", appId)
-        .add("parentClusterId", parentClusterId).toString();
+        .add("parentClusterId", parentClusterId).add("comments", comment).toString();
```
Comment (nobodyiam) [inline apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/entity/Cluster.java:83]:
```suggestion
        .add("parentClusterId", parentClusterId).add("comment", comment).toString();
```

**Review by nobodyiam — COMMENTED** [review]
This looks fantastic! Below are a few comments for your consideration. Additionally, please update the CHANGES.md file to inform users about this new feature.

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**